app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -2298,6 +2299,27 @@ class PluginApi {
     addSidebarPanel(func);
   }
 
+  /**
+   * EXPERIMENTAL. Do not use.
+   * Support for adding links to specific admin sidebar sections.
+   *
+   * This is intended to replace the admin-menu plugin outlet from
+   * the old admin horizontal nav.
+   *
+   * ```
+   * api.addAdminSidebarSectionLink("root", {
+   *   name: "unique_link_name",
```

Comment:
Just curious, does the name need to be snake case or can it be any string?

Reply:
I am pretty sure it can be anything, it's just used here in SidebarSectionLink as unique identifiers for testing/querying link items:

https://github.com/discourse/discourse/blob/18f8ebb2eb30ae50ff911bbb1411cc7c2581d63f/app/assets/javascripts/discourse/app/components/sidebar/section-link.hbs#L1-L13

Technically nothing bad would happen if they gave a duplicated name, but I still prepend stuff onto it just in case.